articles/openshift/howto-deploy-java-eap-app.md

```diff
@@ -0,0 +1,384 @@
+
+The application is a stateful application that stores information on the HTTP Session. It makes use of the JBoss EAP clustering capabilities and uses the following Jakarta EE 8 and Microprofile 4.0 technologies:
+* Jakarta Server Faces
+* Enterprise Java Beans
+* Java Persistence
+* Microprofile Health
+
+> [!IMPORTANT]
+> This article uses a Microsoft SQL Server docker image running on a Linux container on Red Hat OpenShift. Before choosing to run a SQL Server container for production use cases, please review the [support policy for SQL Server Containers](https://support.microsoft.com/help/4047326/support-policy-for-microsoft-sql-server) to ensure that you are running on a supported configuration.
+
+> [!IMPORTANT]
+> While ARO is jointly engineered, operated, and supported by Red Hat and Microsoft to provide an integrated support experience, the software you run on top of ARO, including that described in this article, is subject to its own support and license terms. For details about support of ARO, see [Support lifecycle for Azure Red Hat OpenShift 4](../support-lifecycle.md). For details about support of the software described in this article, see the main pages for that software as listed in the article.
+> * For support for JBoss Enterprise Application Platform, please go to [Red Hat JBoss Enterprise Application Platform](https://www.redhat.com/en/technologies/jboss-middleware/application-platform).
+
+## Prerequisites
+
+> [!NOTE]
+> Azure Red Hat OpenShift requires a minimum of 40 cores to create and run an OpenShift cluster. The default Azure resource quota for a new Azure subscription does not meet this requirement. To request an increase in your resource limit, see [Standard quota: Increase limits by VM series](../azure-portal/supportability/per-vm-quota-requests.md). Note that the free trial subscription isn't eligible for a quota increase, [upgrade to a Pay-As-You-Go subscription](../cost-management-billing/manage/upgrade-azure-subscription.md) before requesting a quota increase.
+
+1. Prepare a local machine with Unix-like operating system installed (for example, Ubuntu, macOS).
+1. Install a Java SE implementation (for example, [AdoptOpenJDK OpenJDK 11 LTS/OpenJ9](https://adoptopenjdk.net/?variant=openjdk11&jvmVariant=openj9)).
+1. Install [Maven](https://maven.apache.org/download.cgi) 3.6.3 or higher.
+1. Install [Docker](https://docs.docker.com/get-docker/) for your OS.
+1. Install [Azure CLI](/cli/azure/install-azure-cli) 2.29.2 or later.
+1. Clone the code for this demo application on your local system. The demo application is on [GitHub](https://github.com/yersan/todo-list).
```

Comment:
I think `todo-list` should appear in the text. Maybe change "Clone the code for this demo application on your local system." to "Clone the code for this demo application (todo-list) on your local system."

Also rather than "on your local system" maybe "to your local system"